pkg/volume/volume.go

```diff
@@ -193,6 +193,13 @@ type BulkVolumeVerifier interface {
 	BulkVerifyVolumes(volumesByNode map[types.NodeName][]*Spec) (map[types.NodeName]map[*Spec]bool, error)
 }
 
+type Expander interface {
+	// ExpandVolume expands the volume device via cloudprovider api call
+	ExpandVolumeDevice(newSize resource.Quantity, oldSize resource.Quantity) error
```

Comment:
I would return also the actual new size of the volume. Some storage backends might round the value to the nearest GiB and you should know the actual value.

Reply:
ack

Reply:
still missing return of actual new size of the volume